test/integration/kubernetes_deploy_test.rb

```diff
@@ -1006,4 +1006,21 @@ def test_raise_on_yaml_missing_kind
       "  datapoint: value1"
     ], in_order: true)
   end
+
+  def test_hpa_can_be_successful
+    skip if KUBE_SERVER_VERSION < Gem::Version.new('1.8.0')
```

Comment:
Your PR description says `The downside of using v2beta is no support in 1.8.`; did you mean 1.7? If that's true, our unofficial deprecation policy (following GKE) would allow us to drop 1.7 at this point, and then the hpa could be folded into `hello-cloud`, as could the cronjob tests. If that isn't the case (i.e. the conditions aren't present in a version we need to support), don't we need to have alternate success/failure conditions in the code itself rather than here, to avoid breaking deploys for people with hpas on that version?

Reply:
It was added in 1.8 and I've updated the description. What does dropping support for 1.7 mean past not running 1.7 CI?